apps/grc-platform/backend/internal/risk/handler/escalation_job_test.go

```
@@ -0,0 +1,105 @@
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/wso2-open-operations/grc-tools/apps/grc-platform/backend/internal/shared/privilege"
+)
+
+func postToEscalationRun(t *testing.T, h *escalationJobHandler, grants map[string]bool) *httptest.ResponseRecorder {
+	t.Helper()
+	req := httptest.NewRequest(http.MethodPost, "/api/v1/risks/escalations/run", nil)
```

golangci-lint (2.12.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)